code/modules/antagonists/brother/brother.dm

```diff
@@ -63,6 +63,11 @@
 		flashed.balloon_alert(source, "[flashed.p_their()] mind is vacant!")
 		return
 
+	for(var/datum/objective/assassinate/assassinate_objective as anything in source.mind.get_all_objectives())
+		if(flashed == assassinate_objective.target.current)
```

Comment:
This is either 
- An improper use of `as anything` (there will be more than assassination objectives in all objectives of a mob) 

Or
- Something that doesn't even need to be limited to assassination objectives (So it works with all similar objectives, IE Maroon, Mutiny, Sacrifice)

Reply:
Didn't occur to me that `target` was set on the objective level. I believe it is the latter.